docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.8"
```

Comment:
I get this warning: `WARN[0000] path/to/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion`